### PR TITLE
rename subchangefeed info to task status

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
Uniform terminology：rename subchangefeed info to task status

### CODE REVIEW

**Comment by suzaku** [comment]
This is confusing. In the original design, processor is a role that handles "sub change feed", now the thing being processed is called processor.

**Comment by leoppro** [comment]
> This is confusing. In the original design, processor is a role that handles "sub change feed", now the thing being processed is called processor.

but now, one processor is one subchangefeed. the name of subchangefeed is long-winded and easy to confuse with changefeed

**Comment by suzaku** [comment]
I would prefer a name like "Task", it's something to be assigned and processed.

**Comment by leoppro** [comment]
I don't want to add more concept or terminology, I think processor is fine here.
what do you think about this? @amyangfei

**Comment by amyangfei** [comment]
IMO processor acts more like a role, and it handles some kind of job. Rename `SubChangeFeed` to a new name such as `Task` does not introduce a new concept.
I prefer to rename it to a short, simple name, like `Task`, `Feed`

**Comment by leoppro** [comment]
ok，I will rename it to Task

**Comment by leoppro** [comment]
PTAL again @amyangfei @suzaku

**Comment by suzaku** [comment]
@leoppro Pls resolve the conflicts.

**Comment by leoppro** [comment]
@suzaku fixed

File: cdc/processor.go [hunk 1]
```diff
@@ -222,15 +222,15 @@ func NewProcessor(pdEndpoints []string, changefeed model.ChangeFeedDetail, chang
 		ddlPuller:     ddlPuller,
 
 		tsRWriter:    tsRWriter,
-		subInfo:      tsRWriter.GetSubChangeFeedInfo(),
+		info:         tsRWriter.GetTaskStatus(),
```
Comment (amyangfei) [inline cdc/processor.go]: s/info/status
is `status` better here?

Comment (leoppro) [reply cdc/processor.go]: yes,updated

File: cdc/roles/storage/etcd.go [hunk 2]
```diff
@@ -219,12 +219,12 @@ func (rw *ProcessorTsEtcdRWriter) ReadGlobalResolvedTs(ctx context.Context) (uin
...
-func (rw *ProcessorTsEtcdRWriter) GetSubChangeFeedInfo() *model.SubChangeFeedInfo {
-	return rw.info
+// GetTaskStatus returns the in memory cache of *model.TaskStatus stored in ProcessorTsEtcdRWriter
+func (rw *ProcessorTsEtcdRWriter) GetTaskStatus() *model.TaskStatus {
+	return rw.taskStatus
 }
 
-// OwnerSubCFInfoEtcdWriter encapsulates SubChangeFeedInfo write operation
+// OwnerSubCFInfoEtcdWriter encapsulates TaskStatus write operation
 type OwnerSubCFInfoEtcdWriter struct {
```
Comment (amyangfei) [inline cdc/roles/storage/etcd.go]: s/OwnerSubCFInfoEtcdWriter/OwnerTaskStatusEtcdWriter

File: cmd/ctrl.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ const (
 	CtrlQueryCfs = "query-cf-list"
 	// query capture list
 	CtrlQueryCaptures = "query-capture-list"
-	// query subchangefeed replication status
+	// query processor replication status
 	CtrlQuerySubCf = "query-sub-cf"
```
[the remark this answers is not in the document]
Comment (leoppro) [reply cmd/ctrl.go]: I think is not necessary, this ctrl is a temporary user interface for developers and testers, we are designing a new `ticdc-ctl` to replace it.

File: cdc/roles/storage/etcd.go (hunk 2 above)
Comment (leoppro) [reply cdc/roles/storage/etcd.go]: updated

**Pull request merged**